src/Pages/Payment/Payment.jsx

```diff
@@ -1,176 +0,0 @@
-                  />
-
-                  <Box>
-                    <Heading
-                      className={PAYMENT.Headings}
-                      fontSize={["sm", "md", "lg"]}
-                    >
-                      Credit/ Debit/ ATM Card
-                    </Heading>
-                    {/* <Text> major cards are supported</Text> */}
-                  </Box>
-                </Flex>
-              </Flex>
-            </CardHeader>
-          </Card>
-
-          <Card
-            maxW="md"
-            bg={"white"}
-            color={"black"}
-            marginTop={"10px"}
-            className={PAYMENT.card}
-            onClick={() => handleChange("wallet")}
-          >
-            <CardHeader>
-              <Flex spacing="4">
-                <Flex flex="1" gap="4" alignItems="center" flexWrap="wrap">
-                  <Avatar
-                    className={PAYMENT.Avatarmain}
-                    name="Mobile Wallet"
-                    src="https://zoomcar-assets.zoomcar.com/images/original/1317404625e385185d23ea43a0c34569d6432d09.png?1632130014"
-                  />
-
-                  <Box>
-                    <Heading
-                      className={PAYMENT.Headings}
-                      fontSize={["sm", "md", "lg"]}
-                    >
-                      Mobile Wallet
-                    </Heading>
-                    {/* <Text>All major wallets are supported</Text> */}
-                  </Box>
-                </Flex>
-              </Flex>
-            </CardHeader>
-          </Card>
-
-          <Card
-            maxW="md"
-            bg={"white"}
-            color={"black"}
-            marginTop={"10px"}
-            className={PAYMENT.card}
-            onClick={() => handleChange("netbank")}
-          >
-            <CardHeader>
-              <Flex spacing="4">
-                <Flex flex="1" gap="4" alignItems="center" flexWrap="wrap">
-                  <Avatar
-                    name="Net Banking"
-                    className={PAYMENT.Avatarmain}
-                    src="https://zoomcar-assets.zoomcar.com/images/original/4dd051886e5403aafb9725b7d2ab42c80f9b1df6.png?1584453529"
-                  />
-
-                  <Box>
-                    <Heading
-                      className={PAYMENT.Headings}
-                      fontSize={["sm", "md", "lg"]}
-                    >
-                      Net Banking
-                    </Heading>
-                  </Box>
-                </Flex>
-              </Flex>
-            </CardHeader>
-          </Card>
-        </div>
-
-        {/* second div starts here */}
-
-        <div>
-          {pageTrue === "upi" ? (
-            <Active />
-          ) : pageTrue === "card" ? (
-            <Credit />
-          ) : pageTrue === "netbank" ? (
-            <Banking />
-          ) : pageTrue === "wallet" ? (
-            <Wallet />
-          ) : null}
```

Comment:
can we move this whole logic inside a function and then call that function here?